plugins/modules/idrac_secure_boot.py

```diff
@@ -0,0 +1,399 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell OpenManage Ansible Modules
+# Version 9.6.0
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: idrac_secure_boot
+short_description: Import secure boot certificate.
+version_added: "9.6.0"
+description:
+  - This module allows to import the secure boot certificate.
+extends_documentation_fragment:
+  - dellemc.openmanage.idrac_x_auth_options
+options:
+  import_certificates:
+    type: bool
+    description:
+        - Import all the specified key certificates.
+        - When I(import_certificates) is C(true) either of I(platform_key) or I(KEK) or I(database) or I(disallow_database) is required.
+  platform_key:
+    type: path
+    description:
+      - The absolute path of the Platform key certificate file for UEFI secure boot.
+  KEK:
+    type: list
+    elements: path
+    description:
+      - A list of absolute paths of the Key Exchange Key (KEK) certificate file for UEFI secure boot.
+  database:
+    type: list
+    elements: path
+    description:
+      - A list of absolute paths of the Database certificate file for UEFI secure boot.
+  disallow_database:
+    type: list
+    elements: path
+    description:
+      - A list of absolute paths of the Disallow Database certificate file for UEFI secure boot.
+  restart:
+    type: bool
+    default: false
+    description:
+      - Secure boot certificate import operation requires a server restart. This parameter provides an option to restart the server.
+      - C(true) will restart the server.
```

Comment:
C(ForceRestart) forcefully reboots the host system.           
C(GracefulRestart) gracefully reboots the host system.

Reply:
Kept as below:

- C(true) restarts the server.
- C(false) does not restart the server.